package/MDAnalysis/topology/guessers.py

```diff
@@ -225,6 +226,16 @@ def guess_atom_element(atomname):
         return no_symbols
 
 
+def guess_atom_element_from_mass(mass_guess, tol=0.1):
+    for element, mass in tables.masses.items():
+        if abs(mass - mass_guess) < tol:
+            return element
+    raise ValueError(f'the molecular weight: {mass_guess} amu, is not valid')
```

Comment:
I think other guessers don't raise Errors but instead warn and return empty/default values